client/js/components/map/admin/AdminLayerListItem.vue

```diff
@@ -30,119 +30,118 @@
+      <span
+        class="font-size-smaller"
+        v-if="false === layer.attributes.isEnabled">
+        <br>{{ Translator.trans('explanation.gislayer.useas.invisible') }}
+      </span>
+      <span
+        class="font-size-smaller"
+        v-if="layer.attributes.isPrint">
+        <br>{{ Translator.trans('explanation.gislayer.useas.print') }}
+      </span>
+    </div><!--
+            Show this Stuff (Visibility-group / show initially on load) only for layer, not for Categories
+ --><template v-if="(layer.type === 'GisLayer') && hasPermission('feature_map_layer_visibility')"><!--
+    --><div class="inline-block w-1/12 text-right">
+        <a
+          v-if="('undefined' !== typeof activeLayer.id || '' !== hoverLayerId) && false === layer.attributes.isBaseLayer && (false === isChildOfCategoryThatAppearsAsLayer)"
+          @click.stop.prevent="toggleVisibilityGroup"
+          @mouseover="setIconHoverState"
+          @mouseout="unsetIconHoverState"
+          :title="hintTextForLockedLayer">
+          <i :class="[iconClass,showGroupableIcon]" />
+        </a>
+      </div><!--
+   --><div class="inline-block w-1/12 text-right">
         <input
           type="checkbox"
           data-cy="adminLayerListItem:toggleDefaultVisibility"
+          :disabled="'' !== layer.attributes.visibilityGroupId || (true === isChildOfCategoryThatAppearsAsLayer)"
           @change.prevent="toggleHasDefaultVisibility"
           :checked="hasDefaultVisibility"
           :class="iconClass">
-      </div>
-      <div
-        v-if="(layer.type !== 'GisLayer' && (false === layer.attributes.layerWithChildrenHidden))"
-        class="w-2/12 text-right">
-        <!-- spacer for groups -->
-      </div>
-      <div class="w-1/12 text-right">
-        <a
-          :href="editLink"
-          data-cy="editLink">
-          <i
-            class="fa fa-pencil u-mr-0_5"
-            aria-hidden="true"
-            :title="Translator.trans('edit')" /><span class="hide-visually">{{ Translator.trans('edit') }}</span>
-        </a>
-        <button
-          class="btn--blank o-link--default u-mr-0_5"
-          data-cy="adminLayerListItem:deleteElement"
-          :title="Translator.trans('delete')"
-          @click.prevent="deleteElement"
-          v-if="childElements.length <= 0">
-          <i
-            class="fa fa-trash"
-            aria-hidden="true" /><span class="hide-visually">{{ Translator.trans('delete') }}</span>
+      </div><!--
+  --></template><!--
+          Show this Stuff for 'special category that looks like an Layer and hides all his children'
+ --><template v-if="(layer.type === 'GisLayerCategory' && layer.attributes.layerWithChildrenHidden)"><!--
+   --><div class="inline-block w-2/12 text-right">
+        <input
+          type="checkbox"
+          data-cy="adminLayerListItem:toggleDefaultVisibility"
+          @change.prevent="toggleHasDefaultVisibility"
+          :checked="hasDefaultVisibility"
+          :class="iconClass">
```

Comment:
```suggestion
        <input
          type="checkbox"
          data-cy="adminLayerListItem:toggleDefaultVisibility"
          :checked="hasDefaultVisibility"
          :class="iconClass"
          @change.prevent="toggleHasDefaultVisibility">
```
If I am not mistaken, these events should be listed at the end of the component.